### ISSUE TITLE
docker-compose up ippserver fails to start

### ISSUE THREAD

**Issue by twatzl** [istopwg/ippsample#201]
When starting the docker-compose file the server fails with the following error:

```
$ docker-compose up ippserver   
Starting ippsample_ippserver_1 ... done
Attaching to ippsample_ippserver_1
ippserver_1  |  * Removing stale PID file /var/run/dbus/pid.
ippserver_1  |  * Starting system message bus dbus
ippserver_1  |    ...done.
ippserver_1  |  * Starting Avahi mDNS/DNS-SD Daemon avahi-daemon
ippserver_1  | Process 24 died: No such process; trying to remove PID file. (/run/avahi-daemon//pid)
ippserver_1  |    ...done.
ippserver_1  | ippserver: Unable to access BinDir "tools": No such file or directory
ippsample_ippserver_1 exited with code 1
```

I just stumbled across this repo, since I was looking for a dummy IPP printer and so I don't really have any idea what could be wrong. As far as I could see all the volumes are mapped. If you need further info for debugging let me know.

Tried it on Fedora 64-bit and on Windows with docker-compose 1.22.0

**Comment by michaelrsweet** [comment]
No idea what may be going wrong, but will try to figure it out...

**Comment by michaelrsweet** [comment]
I've reproduced the issue but still not sure what the issue is.

**Comment by cablespaghetti** [comment]
Mapping a directory to /tools worked for me:

![image](https://user-images.githubusercontent.com/11150054/134678944-2d6ed844-f570-446d-823a-0bbc71ae6ab4.png)

**Comment by michaelrsweet** [comment]
@cablespaghetti Thanks, that certainly does the trick!

[master 45a0dbe] Fix docker support (Issue #201)
